json/Side_Text.txt

```diff
@@ -28412,270 +28412,270 @@
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "うふふふ、そうよね。<%br>そんな顔してるもの。",
-		"tr_text": "",
+		"tr_text": "Eheheh, I see.<%br>I love it when you blush like that.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "11",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "そ、そうなのか？　リーダー……",
-		"tr_text": "",
+		"tr_text": "I-is that true, Leader? Woah...",
 		"fileID": 28
 	},
 	{
 		"eventNo": "12",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "……チラリズムを解するとは<%br>流石だわ、リーダー。",
-		"tr_text": "",
+		"tr_text": "...So you appreciate the subtle sensuality of a<%br>brief flash of white, do you? How cultured of you.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "13",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "そうなんですか？　リーダー。<%br>え、違う？",
-		"tr_text": "",
+		"tr_text": "Is that true, Leader?<%br>She's just jumping to conclusions, isn't she?",
 		"fileID": 28
 	},
 	{
 		"eventNo": "14",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "ま、いいわ……今日の任務でも<%br>たっぷりアピールしちゃうんだから。",
-		"tr_text": "",
+		"tr_text": "Well, that's fine... I'll have plenty of<%br>time to put you under my spell today.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "15",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "セクシーでエレガントなアタシの戦い<%br>よーく見ててよね？",
-		"tr_text": "",
+		"tr_text": "I'll show you my sexy, elegant fighting style.<%br>You will keep your eyes on me, won't you?",
 		"fileID": 28
 	},
 	{
 		"eventNo": "0",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "ブルージーレクイエムさんは<%br>リーダーが好きなんですか？",
-		"tr_text": "",
+		"tr_text": "Bluesy Requiem, do you like Leader?",
 		"fileID": 28
 	},
 	{
 		"eventNo": "1",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "あら、聞かなきゃ分からないほど<%br>適当なアプローチに見えたかしら。",
-		"tr_text": "",
+		"tr_text": "My, my. After all you've seen,<%br>did you really still have to ask?",
 		"fileID": 28
```

Comment:
`Ah, you still don't get it. Maybe if my approach was a bit more formal you'd get it.`